pyproject.toml

```diff
@@ -41,6 +41,7 @@ mypy = ">=0.770"
 flake8 = ">=3.8.1"
 black = ">=19.10b0"
 pytest-cov = ">=2.8.1"
+isort = "^5.10.1"
```

Comment:
why fix the version, and not make it at least this version?

Reply:
Apparently that's just what happens when I `poetry add isort`.  Fixed and relocked.